spec/controllers/home_controller_spec.rb

```diff
@@ -143,6 +143,21 @@
       it "should use the remote_addr when neither is set" do
         controller.send(:rollbar_request_data)[:user_ip].should == '0.0.0.0'
       end
+
+      context "rollbar_user_ip obfuscator" do
+        before do
+          Rollbar.configure do |config|
+            config.user_ip_obfuscator_secret = 'secret'
+          end
+        end
+
+        it "should obfuscate the ip when user_ip_obfuscator_secret is set" do
+          real_ip = '1.1.1.1'
```

Comment:
Thanks for the tests. It'd be nice having more tests with invalid IPs values, so we ensure we don't crash at any point.